layer/USAGE_android.md

```diff
@@ -1,143 +1,151 @@
+the GFXReconstruct layer must be able to detect when the application
+modifies the mapped memory. The layer can be configured to detect memory
+modifications by marking the mapped memory as write protected, triggering
+an access violation when the application writes to the memory. The
+layer then uses a signal handler to intercept the signal generated by the
+access violation, where it removes the write protection, marks the modified
+memory page as dirty, and allows the application to continue.
+
+When running an application in a debugger with the layer enabled, the
+access violations triggered by the layer's memory tracking behavior may
+cause the debugger to break. These debug breaks may be disabled for LLDB with
+the following command:
 ```
 process handle SIGSEGV -n true -p true -s false
 ```
 
-This can be entered manually through the LLDB tab on Android Studio's
+This command may be entered manually through the LLDB tab on Android Studio's
 Debug panel.
 
-It can also be set as a post launch command in the project configuration:
-
-```
-Run > Edit Configurations...
-```
-
-Then choosing:
-
-```
-Debugger > LLDP Post Attach Commands
-```
-
-Where you can click the `+` and add it manually.
-
-## Globally Enabling the Layer
+It may also be set as a post launch command in the project configuration:
+ * Click on "Run" in the menu
+ * Choose "Edit Configurations..."
+ * In the dialog box, select the "Debugger" tab
+ * In the "Debugger" tab, select the "LLDB Post Attach Commands" tab
+ * Click the `+` to add the command to the command list
+ * Click "Apply"
 
-Use ADB to enable the layer for your project by:
+## Enabling the Layer with ADB
+The layer can be enabled through a system property by executing the following
+ADB command:
 
 ```
 adb shell "setprop debug.vulkan.layers 'VK_LAYER_LUNARG_gfxreconstruct'"
 ```
 
-When done, disable the layer using:
+The following command will disable the layer:
 
 ```
 adb shell "setprop debug.vulkan.layers ''"
 ```
 
-## GFXRecon Debug Options
-
-THe GFXReconstruct has multiple options that can be adjusted by using
-Android properties.
-Each property **must** begin with the prefix `debug.gfxrecon.`.
-This may be done by performed by using `adb shell` in the following way:
+## Layer Options
+The GFXReconstruct layer supports the following options, which may be enabled
+with Android system properties.  Each property begins with the prefix `debug.gfxrecon`,
+and can be set through ADB with the following command syntax:
 
 ```
 adb shell "setprop <option> '<value>'"
 ```
 
-For example, to set the log_level to "warning", you would call:
+For example, to set the log_level to "warning", specify:
 
 ```
 adb shell "setprop debug.gfxrecon.log_level 'warning'"
 ```
 
+Options with the BOOL type accept the following values:
+* A case insensitive string value 'true' or a non-zero integer value indicate true.
+* A case insensitive string value 'false' or a zero integer value indicate false.
+
```

Comment:
What happens on all other values, e.g. empty string, or an unrecognized string?

Reply:
I am just going to document the valid values for now, and not the invalid values.  Invalid values should generate a log message, but I need to check on the current state of the settings error handling.